templates/ecs/content/ecs.tf

```diff
@@ -0,0 +1,32 @@
+module "ecs" {
+  source = "github.com/cds-snc/terraform-modules//ecs?ref=v9.2.4"
+
+  # Cluster and service
+  create_cluster = ${{ values.create_cluster }} 
+  cluster_name   = ${{ values.cluster_name | dump }}
+  service_name = ${{ values.service_name | dump }} 
+
+  # Task/Container definition
+  container_image            = ${{ values.container_image | dump }} 
+  task_cpu                   = ${{ values.task_cpu | dump }}  
+  task_memory                = ${{ values.task_memory | dump }}  
+
+  # Scaling
+  enable_autoscaling = true
+
+  # Networking
+  security_group_ids  = ${{ values.security_group_ids | dump }}  
+  subnet_ids          = ${{ values.subnet_ids| dump }}  
+
+  # Forward logs to Sentinel
+  sentinel_forwarder           = true
+  sentinel_forwarder_layer_arn = "arn:aws:lambda:ca-central-1:283582579564:layer:aws-sentinel-connector-layer:100"
```

Comment:
Unfortunately this will end up causing problems for people because the `sentinel_forwarder_layer_arn` value (100 in this case) will always need to be the latest version available in the account.  So far the only way I've found around this is to always lookup the latest layer ARN before applying 😞

Reply:
Thanks Pat! I have removed this code.